app/Filament/Resources/LeaderboardResource.php

```diff
@@ -124,6 +126,11 @@ public static function form(Form $form): Form
 
                         Forms\Components\TextInput::make('Description')
                             ->maxLength(255),
+
+                        Forms\Components\TextInput::make('DisplayOrder')
+                            ->numeric()
+                            ->helperText("If set to -1, the leaderboard will be invisible to regular players.")
```

Comment:
I just wanted to point out that some users have ordered their hidden achievements:
![image](https://github.com/user-attachments/assets/3e833d24-1382-4d8a-97b2-4fa4673fb00a)

It seems like this still works (they can't be reordered, and don't get updated when trying to reorder visible leaderboards. You can also still assign other negative values in the editor.

Maybe this comment needs updated? I think it's probably fine the way it is.

Reply:
I think it's a fair point to consider. I've updated the label to read "If set to less than 0,...".